src/index.tsx

```diff
@@ -44,42 +44,71 @@ function SquircleBackground({
   strokeColor = '#000',
   strokeWidth = 0,
 }: SquircleParams) {
-  const [squircleSize, setSquircleSize] =
+  return (
+    <Rect style={StyleSheet.absoluteFill}>
+      {({ width, height }) => {
+        const squirclePath = getSvgPath({
+          width: width,
+          height: height,
+          cornerSmoothing,
+          cornerRadius,
+          topLeftCornerRadius,
+          topRightCornerRadius,
+          bottomRightCornerRadius,
+          bottomLeftCornerRadius,
+        })
+
+        if (strokeWidth > 0) {
```

Comment:
Personally I'd bring the "general case" first, and the exception after.
Also, `strokeWidth` itself is falsy when 0, so condition could be shortened to `if (!strokeWidth)` or `if (strokeWidth)`. Or `const hasStroke = strokeWidth > 0` or similar, to make it even more evident what's happening.

Reply:
👌